### PR TITLE
Ansible D8 deploy scripts

### PR DESCRIPTION
**Pull request by gchaix**
Moving this over from tag1consulting/ansible-systems#35

### CODE REVIEW

**Comment by gchaix** [comment]
Tested using the following command line on dev:
```
ansible-playbook -i non-production d8-deploy/d8_deploy.yml --extra-vars "site=dev.tag1consulting.com git_deploy_repo=git@github.com:tag1consulting/tag1consulting.com.git git_tag=20171211 drupal_files_source=/var/www/dev.tag1consulting.com/files" --ask-become-pass
```
Two requirements that are not checked in here:
1. an inventory file (`non-production` from tag1consulting/ansible-systems#tag1internal in this case)
2. the `settings.local.php` and `services.yml` files for the site already exist in `/var/www/dev.tag1consulting.com/settings`

File: ansible/ansible.cfg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[defaults]
+# Include 3rd party roles directory as well as local custom roles.
+roles_path = roles:galaxy-roles
+# Disable retry files.
+retry_files_enabled = False
+
+# Enable mitogen.
```
Comment (jeffsheltren) [inline ansible/ansible.cfg]: Can we enable this and require Ansible 2.7 for now?

File: ansible/group_vars/all [hunk 2]
```diff
@@ -0,0 +1 @@
+ansible_python_interpreter: "/usr/bin/python3"
```
Comment (jeffsheltren) [inline ansible/group_vars/all]: Curious what this is for...

File: ansible/ansible.cfg (hunk 1 above)
Comment (gchaix) [reply ansible/ansible.cfg]: Mostly I just didn't want to bloat this PR with all the Mitogen stuff, so I left it out.

File: ansible/group_vars/all (hunk 2 above)
Comment (gchaix) [reply ansible/group_vars/all]: On some Debian systems this is needed or Ansible won't run because it can't find python.

**Pull request merged**